### ISSUE TITLE
Improve support for text selection

### ISSUE THREAD

**Issue by kekukui** [brave/brave-browser#3224]
**1.  `Shift-PgDn`** does not engage text selection mode (which is the standard in most other applications, like WordPad, et cetera)

**2.**  If you right-click on a page, **`Select All`** is not available in the context menu.

**3.**  **`Select All`** is not available in the **Edit** menu.

(Tested on Windows stable branch)

**Comment by srirambv** [comment]
I don't believe these are standard options available on any Chromium browsers. These seem to be specific to Firefox. 

cc: @bbondy  @tomlowenthal @rebron to take a call on implementing this or close as wontfix

**Comment by bsclifton** [comment]
This issue hasn't had an update in a while - it's met the criteria for being stale. The issue is going to be closed for now. If this is still a problem, please leave a comment or re-open the issue. Thanks!
